book/scheduling.md

```diff
@@ -532,11 +531,11 @@ mutations instead of page loads to update its state is called a
 complex web apps, which in turn made browser speed and responsiveness
 more important.
 
-[^when-gmail]: GMail dates from April 2004, [soon after][xhr-history]
-enough browsers finished adding support for the API. The first
+[^when-gmail]: The first
 application to use `XMLHttpRequest` was [Outlook Web Access][outlook],
 in 1999, but it took a while for the API to make it into other
-browsers.
+browsers. GMail dates from April 2004, [soon after][xhr-history]
+enough browsers finished adding support for the API.
```

Comment:
Otherwise weirdly achronological.

Reply:
I disagree - this footnote is keying off of text about GMail so it's appropriate to mention ti first.